src/HeaderCell.tsx

```diff
@@ -172,7 +174,10 @@ export default function HeaderCell<R, SR>({
       // set the tabIndex to 0 when there is no selected cell so grid can receive focus
       tabIndex={shouldFocusGrid ? 0 : tabIndex}
       className={className}
-      style={getCellStyle(column, colSpan)}
+      style={{
```

Comment:
We need to tweak `SortableHeaderCell` also so it uses the same height as parent.

Reply:
Fixed!